marklogic-data-hub-central/ui/e2e/cypress/support/pages/curate.tsx

```diff
@@ -81,6 +81,23 @@ class CuratePage {
         return cy.findByTestId(`${stepName}-edit`);
     }
 
+    runStepInCardView(stepName: string) {
+        return cy.findByTestId(`${stepName}-run`);
+    }
+
+    runInNewFlow(stepName: string) {
+        return cy.findByTestId(`${stepName}-run-toNewFlow`);
+    };
+
+    runExistingFlowsList(stepName: string) {
+        return cy.findByTestId(`${stepName}-run-flowsList`);
+    }
+
+    runStepInExistingFlow(stepName: string, flowName: string) {
+        this.runExistingFlowsList(stepName).click({force: true});
```

Comment:
I am assuming we are using {force: true} here as a last resort ?

Reply:
@xnikhil08 Yeah the menu was getting covered sometimes causing the test to have trouble locating it, so {force:true} seemed like the best solution to fix that.